src/functionsShellCommandAction.js

```diff
@@ -10,15 +10,23 @@ var serveFunctions = require("./serve/functions");
 var LocalFunction = require("./localFunction");
 var logger = require("./logger");
 var shell = require("./emulator/functionsEmulatorShell");
+var commandUtitls = require("./emulator/commandUtils");
 
 module.exports = function(options) {
   options.port = parseInt(options.port, 10);
+
+  let debugPort = undefined;
```

Comment:
nit: `let debugPort: number | undefined`;

Setting something to `undefined` always feels dirty to me haha

Reply:
Agree but this is `.js` ... so I can't add types.